contracts/test-utils/src/types.ts

```diff
@@ -137,3 +137,38 @@ export interface MatchOrder {
     leftSignature: string;
     rightSignature: string;
 }
+
```

Comment:
Aren't all these just used by the new classes in the `exchange` package? Why not keep them close? Either that or put the new classes in `test-utils`.

Reply:
I'm trying to start aggregating types here, like what we do in `@0x/types. I also agree that the new classes should live here as well, but at present it creates a circular dependency so I put them in `exchange` for now. I'll add a note to the code for future reference.